.github/workflows/build-test-image.yaml

```
@@ -654,3 +711,21 @@
+      matrix:
+        platform:
+          - machine_label: cluster-${{ github.run_id }}-${{ github.run_number }}
+          - machine_label: replication-${{ github.run_id }}-${{ github.run_number }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/cleanup-runner
+        with:
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          project_id: ${{ vars.GCP_PROJECT_ID }}
+          zone: ${{ vars.GCP_ZONE }}
+          instance_label: ${{ matrix.platform.machine_label }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)